helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataWriter.java

```diff
@@ -391,8 +422,8 @@ protected boolean sendRequestToLeader(HttpUriRequest request, int expectedRespon
       }
     } catch (IOException e) {
       LOG.error(
-          "The forwarded request to leader raised an exception. Uri: {} Current hostname: {} Leader hostname: {}",
-          request.getURI(), _myHostName, leaderHostName, e);
+          "The forwarded request to leader raised an exception. Uri: {} Current hostname: {} ",
+          request.getURI(), _myHostName, e);
```

Comment:
Why is leader's host name removed? This was very useful information in debugging.

Reply:
Because the leader endpoint is already included in `request.getURI()`. Logging an extra copy is redundant.